Alcatraz/Helpers/ATZStyleKit.m

```diff
@@ -99,16 +96,37 @@ + (void)drawText:(NSString*)text withColor:(NSColor *)color centeredInRect:(NSRe
     CGContextAddPath(context, clippingPath);
     CGContextClip(context);
 
+    // Text font & style
+    NSFont *font = [NSFont fontWithName:@"HelveticaNeue-Light" size:12];
```

Comment:
Np, I've meant - there's no reason to extract `font` into it's own variable versus just passing it directly down.

In general, it's a bad idea to assign local variables if they can be just used as a function